luaui/Widgets/gui_spectator_hud.lua

```diff
@@ -341,6 +341,10 @@ local function buildUnitDefs()
 	end
 end
 
+local function deleteUnitDefs()
+	unitDefsToTrack = {}
+end
```

Comment:
not sure if this deserves a function (similar to how you wouldn't normally do `function add(a, b) return a+b end`) but it's not a big deal